build/.pa11yci.json

```diff
@@ -9,13 +9,14 @@
     },
     "reporters": [
       "cli",
-      ["json", { "fileName": "./pa11y-ci-results.json" }]
+      ["json", { "fileName": "./pa11y-ci-results.json" }],
```

Comment:
Seems to be no longer required (https://gitlab.com/gitlab-ci-utils/pa11y-ci-reporter-html/-/blob/master/docs/upgrade-guide.md) shall we remove it as well ?

Reply:
"Update the Pa11y CI configuration file as specified below, which will generate both the JSON and HTML reports (as the previous commands did)". I'm gonna test but I thought this was mandatory.

Reply:
Trying it with https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1707/commits/81d78fd90c6143290ed9d6f3a90da07f0d3a3196

Reply:
Seems to work. Good catch!